modules/op-setting-resync-period.adoc

```diff
@@ -0,0 +1,38 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-setting-resync-period_{context}"]
+= Setting the resync period for the pipelines controller
+
+You can configure the resync period for the pipelines controller. Once every resync period, the controller reconciles all pipeline runs and task runs, regardless of events. 
+
+The default resync period is 10 hours. If you have a large number of pipeline runs and task runs, a full reconciliation every 10 hours might consume too many resources. In this case, you can configure a longer resync period.
+
+.Prerequisites
+* You are logged in to your {OCP} cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. In the `TektonConfig` CR, configure the resync period for the pipelines controller, as shown in the following example.
++
+.Example of setting the resync period for the pipelines controller to 24 hours
```

Comment:
Suggestion, so this is up to you - would any of these shorter descriptions still work?
```suggestion
.Example `TektonConfig` CR with 24 hour resync period
```
```suggestion
.Example pipelines controller with 24 hour resync period
```
Or simply
```suggestion
.Example
```

Up to you but I think a lot of the context is already explained in the procedure and the file itself, imo